poc/mechanism/mechanism.py

```diff
@@ -0,0 +1,31 @@
+"""
+Interface for DP mechanisms.
+
+A DP mechanism is responsible for sampling noise and adding noise to data.
```

Comment:
Ok, it's a bit of a lie that we would add noise to the aggregate result. We could be more specific about the VDAF syntax, but when talking about DP mechanisms, I think it'll be useful to be somewaht agnostic about the protocol with we're making DP.

```suggestion
A DP mechanism is responsible for sampling noise and adding it to either a measurement or the aggregate result itself.
```

Reply:
@cjpatton I would avoid mentioning aggregate result, in aggregator-DP policy the noise is added to aggregate share not aggregate result. We either be agonistic and say "adding noise to data", or use the exact term used in VDAF.

Reply:
Agree it should be VDAF agnostic. How about "adding it to the data with the type applicable to the DP mechanism"?